### DISCUSSION TITLE
Contribute on Problem LeetCode Java Solution to help all starting DSA in Java

### DISCUSSION THREAD

**Discussion by manishharit** [community/community#66124] [Programming Help]
### Select Topic Area

Product Feedback

### Body

Hello Everyone,
 I have created a new Repository to prepare Leetcode Solution with multiple approaches so that every one learning DSA can use this as a Resource. 

Fork ->  https://github.com/manishharit/leetcode-solutions

Feel Free to contribute if anyone is intrested:
Fork the repo
[Redacted due to Code of Conduct]
Contribute in Readme.md
Push it to the main branch

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @manishharit ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by queenofcorgis** [reply]
Additionally, I did remove a step in your post as we currently do not allow self-promotion on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
